spyder/preferences/languageserver.py

```diff
@@ -595,8 +606,8 @@ def keyPressEvent(self, event):
             self.parent().reset_btn.setFocus()
         elif key in [Qt.Key_Up, Qt.Key_Down, Qt.Key_Left, Qt.Key_Right]:
             super(LSPServerTable, self).keyPressEvent(event)
-        elif key in [Qt.Key_Escape]:
-            self.finder.keyPressEvent(event)
+        # elif key in [Qt.Key_Escape]:
+        #     self.finder.keyPressEvent(event)
```

Comment:
@andfoy, @ccordoba12  what was this supposed to do? this was generating an error when pressing escape on the Completion Linting Advanced Settings section

Reply:
I don't remember actually

Reply:
Since that was created by @andfoy, please remove it.